lib/web/oidcidp.go

```diff
@@ -89,6 +93,61 @@ func (h *Handler) jwksOIDC(_ http.ResponseWriter, r *http.Request, _ httprouter.
 	return &resp, nil
 }
 
+// thumbprint returns the thumbprint as required by AWS when adding an OIDC Identity Provider.
+// This is documented here:
+// https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html
+func (h *Handler) thumbprint(_ http.ResponseWriter, r *http.Request, _ httprouter.Params) (interface{}, error) {
+	// Dial requires the following address format: host:port
+	cfgPublicAddress := h.cfg.PublicProxyAddr
```

Comment:
perhaps use this function? I'm not sure why but someone added it
```suggestion
	cfgPublicAddress := h.PublicProxyAddr()
```